editor.planx.uk/src/@planx/components/shared/Radio/BasicRadio.tsx

```diff
@@ -23,7 +23,13 @@ const BasicRadio: React.FC<Props> = ({
     onChange={onChange}
     control={<Radio variant={variant} />}
     label={title}
-    sx={variant === "default" ? { pb: 1 } : {}}
+    sx={(theme) => ({
+      mb: variant === "default" ? 1 : 0,
+      alignItems: "flex-start",
+      ".MuiFormControlLabel-label": {
```

Comment:
We can also use `import { formLabelClasses } from "@mui/material/FormControlLabel"` to get this in a more reliable way.

```suggestion
    [`& .${formControlLabelClasses.label}`]: {
```

Reply:
Thanks for this @DafyddLlyr , updated.